alternator/executor.cc

```diff
@@ -59,6 +59,20 @@ logging::logger elogger("alternator-executor");
 
 namespace alternator {
 
+sstring table_status_to_sstring(table_status tbl_status) {
```

Comment:
> nitpick: since this function always returns a string constant, this can return a std::string_view instead of an sstring.
Done

Reply:
> Also, if this function is currently used only in executor.cc, I suggest you make it a "static" function (and the table_status class should be defined in executor.cc, not executor.hh).
This function used _proxy member and that's why it wasn't static. I passed _proxy as parameter and made this function static. Done.

Reply:
Please combine the two patches into one so I can review the final result, instead of seeing something wrong in the first patch and then a fix in a second patch.

Reply:
Done. Combined commits to the single patch

Reply:
btw, is it better and more convenient for you to combine commits after every change in pull request?

Reply:
You don't have to combine all the commits into one, but yes, I prefer not to see commit 2 fixing commit 1 - it makes it harder to review (I don't remember what I reviewed last time, and my comment is still on commit 1), and makes the "git log" later ugly.
In a larger change that can be split into multiple commits on a logical basis, that's even better, but this wasn't the case here.

Reply:
Okay, I understand. Will try to do more convenient for you to review. Thanks.